pkl-cli/pkl-cli.gradle.kts

```diff
@@ -152,12 +158,50 @@ val testStartJavaExecutable by
     doLast { outputFile.get().asFile.writeText("OK") }
   }
 
-tasks.check { dependsOn(testStartJavaExecutable) }
+// Setup `testStartJavaExecutable` tasks for multi-JDK testing.
+val testStartJavaExecutableOnOtherJdks =
```

Comment:
```
./gradlew :pkl-cli:check
```
...
```
> Task :pkl-cli:testStartJavaExecutableJdk18
Pkl 0.28.0-dev+263588b (Linux ..., Java 18.0.2.1)

> Task :pkl-cli:testStartJavaExecutableJdk19
Pkl 0.28.0-dev+263588b (Linux ..., Java 19.0.2)

> Task :pkl-cli:testStartJavaExecutableJdk20
Pkl 0.28.0-dev+263588b (Linux ..., Java 20.0.2)

> Task :pkl-cli:testStartJavaExecutableJdk21
Pkl 0.28.0-dev+263588b (Linux ..., Java 21.0.5)

> Task :pkl-cli:testStartJavaExecutableJdk22
Pkl 0.28.0-dev+263588b (Linux ..., Java 22.0.2)

> Task :pkl-cli:testStartJavaExecutableJdk23
Pkl 0.28.0-dev+263588b (Linux ..., Java 23.0.1)
```

Reply:
Nice!

BTW: `testStartJavaExecutable` is a sanity check; doesn't actually use the Pkl evaluator.